cms/templates/js/show-correctness-editor.underscore

```diff
@@ -0,0 +1,33 @@
+<form>
+<h3 class="modal-section-title" id="show_correctness_label"><%- gettext('Assessment Results Visibility') %></h3>
+<div class="modal-section-content show-correctness">
+    <div role="radiogroup" class="list-fields" aria-labelledby="show_correctness_label">
+        <label class="label">
+            <input class="input input-radio" name="show-correctness" type="radio" value="always" aria-describedby="always_show_correctness_description">
+            <%- gettext('Always show assessment results') %>
+        <p class='field-message' id='always_show_correctness_description'>
+            <%- gettext('When learners submit an answer to an assessment, they immediately see whether the answer is correct or incorrect, and the score received.') %>
+        </p>
```

Comment:
@pomegranited the missing close tag is messing up this HTML (labels are not properly getting associated with radio buttons, number of items isn't working). Can you correct this ASAP and update the sandbox so I can try it out?